account_asset_management/views/account_move.xml

```diff
@@ -3,7 +3,36 @@
     <record id="view_move_form" model="ir.ui.view">
         <field name="name">account.move.form.account.asset.management</field>
         <field name="model">account.move</field>
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='invoice_line_ids']/tree/field[@name='quantity']"
+                position="before"
+            >
+                <field
+                    name="asset_profile_id"
```

Comment:
Why do you still need this in regular view?

Reply:
My first intention was to remove it, but then I thought that if it had originally been done that way it would be for something.
Perhaps it is so that a user without account_manager permissions can enter a purchase invoice even if it is later the account_manager who validates the asset